config/settings.yml

```diff
@@ -92,16 +92,19 @@ edu:
 dependents:
   prefill: true
 
+vre_counseling:
+  prefill: true
+
```

Comment:
Looks like all the other changes in this file were victim to an editor's auto-fixing quotes.

Our [`rubocop.yml`](https://github.com/department-of-veterans-affairs/vets-api/blob/master/.rubocop.yml#L33) enforces singe-quote style.

Reply:
ouch, good catch. Updated this.